recipes/py-fann2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "fann2" %}
+{% set version = "1.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0f2ae9e14630d4d652ffb58f9239ca488e5c15c38e9302f204c24d8c2f7cf0d8" %}
+
+package:
+  name: 'py-{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url:  https://github.com/FutureLinkCorporation/{{ name }}/archive/{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  # Skip the win build because fann2 doesn't build there yet
```

Comment:
Would you mind removing this comment? When you commit please use `[ci skip]` `[circle skip]` ` [travis skip]` to skip the CIs. https://github.com/conda-forge/staged-recipes/issues/1148